### ISSUE TITLE
Strange errors on django 1.8

### ISSUE THREAD

**Issue by M1hacka** [travis-ci/travis-ci#9258]
Hi. I use travis to test [OpenSource library](https://github.com/M1hacka/django-pg-bulk-update/commits/master) using python and django.
I get strange error using django 1.8 and python 3.4+:
```
[GCC 4.8.4]
/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
Creating test database for alias 'default'...
Traceback (most recent call last):
  File "runtests.py", line 22, in <module>
    failures = test_runner.run_tests(["tests"])
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/test/runner.py", line 210, in run_tests
    old_config = self.setup_databases()
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/test/runner.py", line 166, in setup_databases
    **kwargs
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/test/runner.py", line 370, in setup_databases
    serialize=connection.settings_dict.get("TEST", {}).get("SERIALIZE", True),
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/db/backends/base/creation.py", line 368, in create_test_db
    test_flush=True,
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/core/management/__init__.py", line 120, in call_command
    return command.execute(*args, **defaults)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(*args, **options)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/core/management/commands/migrate.py", line 93, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/db/migrations/executor.py", line 19, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/db/migrations/loader.py", line 47, in __init__
    self.build_graph()
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/db/migrations/loader.py", line 174, in build_graph
    self.load_disk()
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/db/migrations/loader.py", line 86, in load_disk
    six.moves.reload_module(module)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/utils/six.py", line 90, in __get__
    result = self._resolve()
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/utils/six.py", line 158, in _resolve
    module = _import_module(self.mod)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/django/utils/six.py", line 80, in _import_module
    __import__(name)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/imp.py", line 33, in <module>
    PendingDeprecationWarning, stacklevel=2)
PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
```
[This source](https://github.com/tox-dev/tox/issues/436) says, I need to update virtualenv as a workaround, but it's strange that fail is on django 1.8 only. Can you suggest, how to fix it?

**Comment by BanzaiMan** [comment]
Could you try reproducing the problem with [our Docker images](https://docs.travis-ci.com/user/common-build-problems/#Running-a-Container-Based-Docker-Image-Locally)?

**Comment by M1hacka** [comment]
Yes, I've installed [python docker image](travisci/ci-garnet:packer-1512502276-986baf0) and reproduced [this test](https://travis-ci.org/M1hacka/django-pg-bulk-update/jobs/348832945)

**Comment by BanzaiMan** [comment]
That’s good news. If you can identify the problem, perhaps we can fix it.

**Comment by M1hacka** [comment]
I've investigated a little bit more, and found, that I've made pip installing x.x.0 django version instead of latest minor release. That's why I had no problems locally, but travis failed. It seems to be a bug in django 1.8.0 release that was fixed later.
P. s. for futher readers, I've used this way to install latest minor version from env variable
```
install:
  - pip install -q Django==$DJANGO.*
```
